tests/markdown/list/tab.md

```diff
@@ -0,0 +1,23 @@
+* Text
+
+ 	[title](link)
```

Comment:
That's not the code from the issue:

> * There are no trailing spaces after the word "Text" on the first line.
> * The second line is empty. (no spaces)
> * The third line starts with a space then a tab.

Reply:
Please pay attention to the kind of whitespace used in the bug reports. The remark update fixes a lot of things related to tabs.

Reply:
Sorry what is the difference? I think that meets above three conditions and also happens syntax error on current stable Prettier:

**Prettier 2.0.5**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAqABAFTgDxgHSkPXxgG0YBLGAGzgF0AKGyqAawEpCQAaECAA5VoAZ2SgAhgCcpEAO4AFaQjEoJNORICeYvgCMpEsGzgwAygKOsA5shhSArnD4ALGAFsaAdRfU4IyzA4MxVqSgA3ai1kcBFdEFYROCkYBUNrdwlkADN1JL4AKxEcACFDY1MzCXc4ABlWOBy85xAinDMbOgBFBwh4Jpp8kEspJKkYzKk2ABN5KF5hqVYYL0ppmBdkAA4ABj4BWSSvQwEYg-9k8Ma+AEde+DTBVRAJEQBaKDg4ae+FqTg7pR-mkJBkskhcoMWkl3JQ7I5oZ04D0+o0Ic0+DAJHpVutNkgAEyYwyUFhQawAYQg7kyMX8AFYFg4kphsapIUNwk4AJJQH6wMxgJZCACCfLMMC0dAGSQAvrKgA)
```sh
--parser markdown
```

**Input:**
```markdown
* Text

 	[title](link)

```

**Output:**
```markdown
SyntaxError: Invalid regular expression: /^\s{0,3}[/: Unterminated character class
```

Reply:
I see you figured this out.